packages/@aws-cdk/aws-lambda-event-sources/test/test.kafka.ts

```diff
@@ -80,6 +80,42 @@ export = {
         ],
       }));
 
+      test.done();
+    },
+    'undefined secret'(test: Test) {
```

Comment:
Would prefer if you changed the test case `default` to not have a secret. And updated this test case to include the secret.

Same applies for the self managed kafka test suite.

Reply:
Flipped these around